gnomad_constraint/resources/resource_utils.py

```diff
@@ -116,6 +123,30 @@ def get_training_dataset(
     )
 
 
+def get_models(
+    model_type: str,
+    version: str = CURRENT_VERSION,
+    genomic_region: str = "autosome_par",
+    test: bool = False,
+) -> str:
+    """
+    Return path to a pickle file that saves the expression of model.
+
+    :param model_type: The type of model. One of "plateau_models", "coverage_model". Default is None.
```

Comment:
```suggestion
    :param model_type: The type of model. One of "plateau", "coverage". Default is None.
```
(if you take the suggestion above)